source/GameInterface/Services/CultureObjects/Handlers/CultureObjectHandler.cs

```diff
@@ -0,0 +1,59 @@
+﻿using Common.Logging;
+using Common.Messaging;
+using Common.Network;
+using Common.Util;
+using GameInterface.Services.BasicCultureObjects.Messages;
+using GameInterface.Services.ObjectManager;
+using Serilog;
+using TaleWorlds.CampaignSystem;
+using TaleWorlds.Core;
+
+namespace GameInterface.Services.BasicCultureObjects.Handlers
+{
+    internal class CultureObjectHandler : IHandler
```

Comment:
Can you rename this to CultureObjectLifetimeHandler and create a test? MBObjectManager is the fallback in our IObjectManager so we may not even need this class but if that doesn't work we do need this